src/test/kotlin/car/business/CarsTest.kt

```diff
@@ -0,0 +1,66 @@
+package car.business
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.ValueSource
+import kotlin.random.Random
+
+class CarsTest() {
+    @ParameterizedTest
+    @ValueSource(ints = [0, 1, 2, 3])
+    fun `랜덤값이 4 미만일경우 움직이면은 안된다`(rand: Int) {
+        // given
+        val amount = 5
+        val random = createMockNextIntRandom(rand)
+        val cars = Cars(amount, random)
```

Comment:
재사용하지 않는 지역 변수의 수는 최소화하는 것이 좋아요. 또는 이름 붙인 인자를 사용하면 어떨까요?

```suggestion
        val cars = Cars(5, createMockNextIntRandom(rand))
```